interface/weno/rxlogmanager.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Rx\Weno\Container;
+
+$container = new Container();
+$log_review = $container->getLogproperties();
+$wenoProperties = $container->getTransmitproperties();
+$logurlparam = $log_review->logReview();
+$provider_info = $wenoProperties->getProviderEmail();
+
+if ($logurlparam == 'error') {
+    echo xlt("Cipher failure check encryption key");
+    exit;
+}
+
+$url = "https://test.wenoexchange.com/en/EPCS/RxLog?useremail=";
+
+$urlOut = $url . $provider_info['email'] . "&data=" . urlencode($logurlparam);
```

Comment:
need to urlencode() the $provider_info['email'] variable

Reply:
same reason as previously stated

Reply:
state a comment above this line to make it clear can not use urlencode() on the $provider_info['email'] (or else it will likely get added in the future on a code walk through)

Reply:
@juggernautsei , still need to address this